play-ws-standalone/src/main/scala/play/api/libs/ws/StandaloneWSRequest.scala

```diff
@@ -138,7 +146,17 @@ trait StandaloneWSRequest {
     val newHeaders = headers.toList.flatMap { param =>
       param._2.map(p => param._1 -> p)
     } ++ hdrs
-    withHttpHeaders(newHeaders: _*)
+
+    // Content-Type is single valued:
+    // https://tools.ietf.org/html/rfc7231#section-3.1.1.5
+    //
+    // So we should undo the append made above.
```

Comment:
Scala doc says "Returns this request with the given headers, preserving the existing ones" when using `addHttpHeaders`, but here we're throwing away the original Content-Type value.